Lamouresparus/Asa
# Asa 28 implement profile screens UI

### Tracked by [JIRA-28](https://loveotudor.atlassian.net/browse/ASA-28)
 
### This PR adds

A UI to display a User's Profile

### Considerations and implementation

We have added a User Interface to display the User Profile which includes courses offered, Institution, Department, Bio, and name.
This implementation also contains a UI for the user to edit his Institution, Bio, Department, and his/her profile picture.

### How to Test

1. Launch the App
2. On the home screen, click on the "Profile Icon": should see a new screen displaying the user's profile.

### Test Added
N/A
